lib/Target/HSACO/HSACOTranslation.cpp

```diff
@@ -67,6 +67,10 @@ initialize_module(llvm::Module *module, const std::string &triple,
   std::string error;
   auto target =
       llvm::TargetRegistry::lookupTarget(module->getTargetTriple(), error);
+  if (target == nullptr) {
+    std::cout << "LookupTarget fail: " << error << std::endl;
+    return nullptr;
+  }
```

Comment:
Without this, triton-translate crash with segfault on line 78 (`target->createTargetMachine`) if triple is not set properly.